examples/deepcopy-gen/output_tests/slices/zz_generated.go

```diff
@@ -78,6 +78,19 @@ func (in *Ttest) DeepCopyInto(out *Ttest) {
 		*out = make([]string, len(*in))
 		copy(*out, *in)
 	}
+	if in.StringPtr != nil {
+		in, out := &in.StringPtr, &out.StringPtr
+		*out = make([]*string, len(*in))
+		for i := range *in {
+			if (*in)[i] == nil {
+				(*out)[i] = nil
+			} else {
+				(*out)[i] = new(string)
+				x := *(*in)[i]
+				(*out)[i] = &x
```

Comment:
If you want to keep the new (per Tim) then these two lines should just be this one: ```*(*out)[i] = *(*in)[i]```

Reply:
fixed